### ISSUE TITLE
paymentRequestWithCardForm on iOS with 9.0.0 throwing exception

### ISSUE THREAD

**Issue by jonasblumer** [tipsi/tipsi-stripe#746]
## Before I have submitted the issue 
[x] I have read an [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html)  
[x] I know that for an iOS I need to install pods because I've read the [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html#cocoapods-ios)  
[x] I have read a [linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html) and checked that everything is OK like in [manual linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html#manual)  
[x] I know that before using `tipsi-stripe` I need to set options for my app as described in [usage guide](https://tipsi.github.io/tipsi-stripe/docs/usage.html)  

## The problem
Since update 9.0.0 ios can't call paymentRequestWithCardForm

## Environment
* `tipsi-stripe` version: 9.0.0
* Last `tipsi-stripe` version where the issue was not reproduced (if applicable): 
* iOS or Android: iOS
* OS version: 14.4
* React-Native version: 0.63.2

```Exception '*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]' was thrown while invoking paymentRequestWithCardForm on target StripeModule ```

## Screenshots, GIFs (Must to have)
![image](https://user-images.githubusercontent.com/5373506/112981766-ed0c7480-915b-11eb-9540-615be9ba7e69.png)

## Code To Reproduce Issue (Good To Have)
Just default example code will create this issue

**Comment by tarandeepmatharu** [comment]
same issue is happening with me

**Comment by nicholasmarais1158** [comment]
As a workaround, provide a complete theme.

```
stripe.paymentRequestWithCardForm({
    theme: {
        primaryBackgroundColor: 'white',
        secondaryBackgroundColor: 'white',
        primaryForegroundColor: 'black',
        secondaryForegroundColor: 'black',
        accentColor:'blue',
        errorColor:'red',
    },
});
```

**Comment by minhas-tech** [comment]
@nicholasmarais1158 After adding this error is not showing up, but IOS app crashes without any error. I am using latest xcode 12.5

**Comment by nicholasmarais1158** [comment]
> @nicholasmarais1158 After adding this error is not showing up, but IOS app crashes without any error. I am using latest xcode 12.5

@minhas-tech If you **don't** provide a `theme` to `stripe.paymentRequestWithCardForm()` do you get the error stated in this issue? If not, the problem you're facing is potentially not related.

**Comment by minhas-tech** [comment]
> > @nicholasmarais1158 After adding this error is not showing up, but IOS app crashes without any error. I am using latest xcode 12.5
> 
> @minhas-tech If you **don't** provide a `theme` to `stripe.paymentRequestWithCardForm()` do you get the error stated in this issue? If not, the problem you're facing is potentially not related.

Yes If I don't provide a theme, I got error as mentioned above. But when I provide the theme. IOS app crash when calling this function. And that crash didn't give any error. Just a silent crash.

**Comment by nicholasmarais1158** [comment]
> > > @nicholasmarais1158 After adding this error is not showing up, but IOS app crashes without any error. I am using latest xcode 12.5
> > 
> > 
> > @minhas-tech If you **don't** provide a `theme` to `stripe.paymentRequestWithCardForm()` do you get the error stated in this issue? If not, the problem you're facing is potentially not related.
> 
> Yes If I don't provide a theme, I got error as mentioned above. But when I provide the theme. IOS app crash when calling this function. And that crash didn't give any error. Just a silent crash.

To not get any error at all is very peculiar.

Please could you make sure you've done a fresh `npm install` (or yarn), fresh `pod install`, cleaned and rebuild your xCode project and restarted your packager. Then try again.

If the problem persists, please share some code and details of your setup.

**Comment by minhas-tech** [comment]
<img width="1440" alt="Screenshot 2021-05-06 at 3 05 00 PM" src="https://user-images.githubusercontent.com/80686316/117281646-50857280-ae7d-11eb-846d-b63070f369a4.png">
I got this error, that's why my app is crashing, it's due to nil image. This issue has been solved in stripe 21.4 but tipsi still dependent on 21.3.1 . Any suggestion?

<img width="887" alt="Screenshot 2021-05-06 at 2 56 06 PM" src="https://user-images.githubusercontent.com/80686316/117281859-89254c00-ae7d-11eb-864b-f88e17c89f87.png">

**Comment by nicholasmarais1158** [comment]
Great. So your issue is not strictly related to this issue.

You should be able to run `pod update Stripe` to update the dependency. This will override and update the version locked in your Podfile.lock. `tipsi-stripe` "supports" Stripe versions `>= 21.3.1` according to it's podspec.

**Comment by minhas-tech** [comment]
I am still getting the same error, Image nill

**Comment by LukhiKartik** [comment]
Any update?

**Comment by fbartho** [comment]
Closing this ticket, so that new users don't think this project is still active.

Stripe does not want you using this, and you will find pain if you do; Please migrate to the official [@stripe/stripe-react-native](https://github.com/stripe/stripe-react-native) package, for your user's safety, and your developer's sanity!

See more: 
- https://github.com/tipsi/tipsi-stripe/issues/779
- https://github.com/tipsi/tipsi-stripe/issues/842
